src/DefaultUnleash.php

```diff
@@ -115,14 +116,14 @@ public function isEnabled(string $featureName, ?Context $context = null, bool $d
         return false;
     }
 
-    public function getVariant(string $featureName, ?Context $context = null, ?Variant $fallbackVariant = null): Variant
+    public function getVariant(string $featureName, ?Context $context = null, ?ResolvedVariant $fallbackVariant = null): ResolvedVariant
```

Comment:
> Or can we rename them so there's no BC break?
So I tried that, it results in a lot of internal changes and make the whole PR ugly to read, I don't think either of us liked it much.

I'd really rather have this out sooner rather than later and we can absolutely make this change in a future PR, so I've made this rolled this back to the original Variant DTO and provided default parameters to the places in DefaultProxyUnleash that return this.